src/components/NoServicesList.jsx

```diff
@@ -2,18 +2,19 @@ import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
 import { Title, Text } from 'cozy-ui/react/Text'
 import { Media, Img, Bd } from 'cozy-ui/transpiled/react/Media'
+import { queryConnect } from 'cozy-client'
 
 import CandidateServiceTile from 'components/CandidateServiceTile'
 import CandidateCategoryTile from 'components/CandidateCategoryTile'
 import AddServiceTile from 'components/AddServiceTile'
 import candidatesConfig from 'config/candidates'
 import ArrowIllustration from 'assets/images/drawing-arrow-up.svg'
 
-export const NoServicesList = ({ t }) => (
+export const NoServicesList = ({ t, konnectorSuggestions }) => (
   <>
     <div className="services-list">
-      {candidatesConfig.konnectors.map(({ slug, name }) => (
-        <CandidateServiceTile key={slug} slug={slug} name={name} />
+      {konnectorSuggestions.data.map(suggestion => (
```

Comment:
Should we test if request has failed / errored?

Reply:
I don't think it's worth displaying an error if we failed to find candidates, since there's nothing the user can do about it.